cosima_cookbook/explore.py

```diff
@@ -941,3 +974,12 @@ def _load_variables(self):
         """
         self.var_selector.set_variables(self.variables)
         self.var_selector._filter_eventhandler(None)
+
+    def data(self):
```

Comment:
This _could_ be a property, but we haven't really made much use of them...

```suggestion
    @property
    def data(self):
```

Reply:
That is a good idea. Makes it look simpler for the user.